packages/kit/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "@module-federation/kit",
+  "version": "1.0.0",
+  "author": "zhanghang <hanric.zhang@gmail.com>",
+  "main": "./index.cjs",
+  "module": "./index.esm.js",
+  "types": "./index.cjs.d.ts",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "exports": {
+    ".": {
+      "import": "./index.esm.js",
+      "require": "./index.cjs.js"
+    },
+    "./runtime": {
+      "import": "./runtime.esm.js",
+      "require": "./runtime.cjs.js"
+    },
+    "./webpack-bundler-runtime": {
+      "import": "./webpack-bundler-runtime.esm.js",
+      "require": "./webpack-bundler-runtime.cjs.js"
+    },
+    "./*": "./*"
+  },
+  "typesVersions": {
+    "*": {
+      ".": [
+        "./index.cjs.d.ts"
+      ],
+      "runtime": [
+        "./runtime.cjs.d.ts"
+      ],
+      "webpack-bundler-runtime": [
+        "./webpack-bundler-runtime.cjs.d.ts"
+      ]
+    }
+  }
```

Hey there, Rodrigo from Valor here.
I've been keeping up with this branch, checking into things for the nextjs plugin.

I've found an issue trying to build all packages and this seems to be missing here. Without it, I was unable to build this package. Sorry if this is unnecessary, you may disregard.

```suggestion
  },
  "dependencies": {
    "@module-federation/runtime": "workspace:*",
    "@module-federation/webpack-bundler-runtime": "workspace:*"
  }
```

---

Sorry for replying later, this package has been removed , and we will export runtime kit from 'runtime-tools' package which is used in rspack .